data/human/culture republic.txt

```diff
@@ -0,0 +1,125 @@
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# "Missions" which add culture events to the spaceport.
+
+#mission "ZZCU cultural holiday template"
+#	minor
+#	source
+#		government "placeholder"
+#		attributes "placeholder"
+#		not attributes "station"
+#	to offer
+#		and
+#			month == placeholder
+#			day >= placeholder
+#			day <= placeholder
+#	on offer
+#		conversation
+#			`placeholder`
+#				decline
+			
+#mission "ZZCU cultural random template"
+#	minor
+#	source
+#		government "placeholder"
+#		attributes "placeholder"
+#		not attributes "station"
+#	to offer
+#		random == 1
+#	on offer
+#		conversation
+#			`placeholder`
+#				decline
+
+
+#############################################
+#      Republic with Syndicate Pairing      #
+#############################################
+
+mission "ZZCU Sad Gamblers -R"
+	minor
+	source
+		government "Republic"
+		attributes "near earth"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success and your attention is drawn as one of them wins a moderate sum in an 12th division win (a day or two worth of standard pay).`
+			`	It's difficult not to pity them as they immediately buy more tickets with the whole value.`
+				decline
+
+#############################################
+#               Republic Only               #
+#############################################
+
+mission "ZZCU Gift Store Interaction"
+	minor
+	source
+		government "Republic"
+		attributes "deep"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`As you wander the spaceport with a snack, waiting for your ship to be refueled, you pass a local souvenir shop and poke your head in to kill a little time. You see a decent looking mug with the text "<source>" printed underneath an attractive, stylized representation of the spaceport you are presently standing in and pick it up to see how it feels. The handle is solid, and not too small for your fingers, and the lip of it looks like it would be comfortable to drink out of.`
+			`	Two young adults wander into the aisle just a couple of meters away. They're clearly from abroad judging by their conversation occurring in some local dialect which is neither local nor one you understand. They're cooing over an item they've found but when one flips it over to look at the price tag there is an immediate exchange of looks and expressions that does not require translation. The item ends up back on the shelf with a giggle and a head-shake.`
+			`	Curious you turn over your own mug and see that it is indeed at least five times what you would expect to pay for a mug, even a good one. Still, t is certainly a novelty...`
```

Comment:
`	Curious you turn` should be `	Curious, you turn`
`Still, t is certainly a novelty...` should be `Still, it is certainly a novelty...`